onnx/backend/test/case/model/__init__.py

```diff
@@ -57,6 +57,12 @@ def collect_testcases():  # type: () -> List[TestCase]
         ('test_vgg19', 'vgg19', 1e-3, 1e-7),
         ('test_zfnet512', 'zfnet512', 1e-3, 1e-7),
     ]
+    if platform.architecture()[0] == '64bit':
```

Comment:
If we are excluding these tests in `onnx/test/test_backend_test.py`, do we need to check for `platform.architecture` here as well?

Reply:
Probably not. Thank you! I reverted this change.